app/states/unified_sqlmodels.py

```
@@ -0,0 +1,1467 @@
+    """Junction table for transaction-person relationships with roles"""
+    __tablename__ = "unified_transaction_persons"
+    
+    id: Optional[int] = Field(default=None, primary_key=True)
+    uuid: str = Field(default_factory=lambda: str(uuid.uuid4()), unique=True, index=True)
+    
+    # Foreign keys
+    transaction_id: int = Field(foreign_key="unified_transactions.id")
+    person_id: int = Field(foreign_key="unified_persons.id")
+    entity_id: Optional[int] = Field(default=None, foreign_key="unified_entities.id")
+    state_id: Optional[int] = Field(default=None, foreign_key="states.id")
+    state_id: Optional[int] = Field(default=None, foreign_key="states.id")
```

Ruff (0.14.7)

Class field `state_id` is defined multiple times

Remove duplicate field definition for `state_id`

(PIE794)